apps/dapp/src/components/Notification/NotificationManager.tsx

```diff
@@ -1,16 +1,54 @@
-import React from 'react';
 import styled from 'styled-components';
 import { useNotification } from 'providers/NotificationProvider';
 import Notification from 'components/Notification/Notification';
+import { useEffect, useState } from 'react';
+
+const AUTOHIDE_TIMER = 1000 * 10;
 
 const NotificationManager = () => {
   const { notifications } = useNotification();
 
+  const { closeNotification } = useNotification();
+
+  // Array indicating which notifications are open.
+  // I know what you're thinking - "couldn't this just
+  // be a boolean because we are always setting the whole
+  // array to either true of false?" Well I tested that,
+  // and it messes up the animation when opening multiple
+  // notifications at once, so let's keep it like this.
+  const [areOpen, setAreOpen] = useState<boolean[]>([]);
+
+  useEffect(() => {
+    // Setting all notifications to be open by default
+    var listTrue = new Array(notifications.length).fill(true);
```

Comment:
nit: let's stick to lets or consts for consistency

Reply:
done